src/js/game/systems/filter.js

```diff
@@ -48,10 +42,11 @@ export class FilterSystem extends GameSystemWithFilter {
     /**
      *
      * @param {Entity} entity
-     * @param {number} slot
-     * @param {BaseItem} item
+     * @param {Object} param0
+     * @param {BaseItem} param0.item
+     * @param {number} param0.extraProgress
      */
-    tryAcceptItem(entity, slot, item) {
+    tryAcceptItem(entity, { item, extraProgress }) {

```

Comment:
Even if more verbose, destructuring is kinda slow, so I'd prefer `tryAcceptItem(entity, item, extraProgress)`